MekHQ/src/mekhq/campaign/stratcon/StratconRulesManager.java

```diff
@@ -2191,6 +2193,35 @@ public static void processScenarioCompletion(ResolveScenarioTracker tracker) {
                     processTrackForceReturnDates(track, campaign);
 
                     track.removeScenario(scenario);
+
+                    if (backingScenario.getStratConScenarioType().isResupply()) {
+                        ResourceBundle resources = ResourceBundle.getBundle("mekhq.resources.Resupply");
+
+                        if (victory) {
+                            campaign.addReport(String.format(resources.getString("convoyRescuedStratCon.text"),
+                                spanOpeningWithCustomColor(MekHQ.getMHQOptions().getFontColorPositiveHexColor()),
+                                CLOSING_SPAN_TAG));
+                        } else {
+                            campaign.addReport(String.format(resources.getString("convoyDefeatedStratCon.text"),
+                                spanOpeningWithCustomColor(MekHQ.getMHQOptions().getFontColorNegativeHexColor()),
+                                CLOSING_SPAN_TAG));
+                        }
+                    } else if (backingScenario.getStratConScenarioType().isLosTech()) {
+                        if (victory) {
+                            int roll = randomInt(10);
+                            StarLeagueCache cache = new StarLeagueCache(campaign, ((AtBContract) mission),
+                                CacheType.TRASH_CACHE.ordinal());
+
+                            // The rumor is a dud
+//                            if (false) { // TODO replace placeholder value
+//                                cache.createDudDialog(track, scenario);
+//                            } else {
+//                                if (Objects.equals(cache.getFaction().getShortName(), "SL")) {
+//                                    cache.createProposalDialog();
+//                                }
+//                            }
```

Comment:
Just to mark

Reply:
You might have noticed a couple of these - they're related to the LosTech portion of this module that will be launched separately. We decided to split implementation in two mid-production, as this allows us to work on LosTech Caches while addressing bugs and feedback with the logistics portion of the module. I should have mentioned this in the PR description. mea culpa.

Reply:
Is this still the operating plan?

Reply:
Yeah, absolutely. It’s for the star league caches I mentioned yesterday